### ISSUE TITLE
Ignore value of property regardless of its type (array of string should be ignored)

### ISSUE THREAD

**Issue by theghostbel** [edvardchen/eslint-plugin-i18next#29]
When I set `ignoreProperty` I expect that all values would be ignored, no matter what type it has. Currently, it seems that only string value is ignored. But when ignoredProperty has an array of strings, then these strings are validated.

My config:
```
ignoreProperty: ['animation']
```

This code works:
```
const iconStyles: SxStyleProp = {
  animation: 'flightMove 4s infinite
};
```

This code fails:
```
const iconStyles: SxStyleProp = {
  animation: ['flightMove 4s infinite', 'flightMove 5s infinite']
};
```

**Comment by edvardchen** [comment]
We cannot ignore  the value's type. Consider the following case:

```js
class A {
  render(){
  	return "hello world";
  }
}
```

Instead we can ignore string arrays. @theghostbel

**Comment by edvardchen** [comment]
I close this and it's ok to reopen it if necessary.
